specification/search/data-plane/Microsoft.Azure.Search.Service/readme.md

```diff
@@ -149,21 +149,12 @@ directive:
     from: "search-request-options"
     to: "request-options"
 
-# Add missing data types to DataType
-- from: swagger-document
-  where: $.definitions.DataType.enum
-  transform: |
-    if (!$.includes("Collection(Edm.String)")) {
-      $.push("Collection(Edm.String)",
-        "Collection(Edm.Int32)",
-        "Collection(Edm.Int64)",
-        "Collection(Edm.Double)",
-        "Collection(Edm.Boolean)",
-        "Collection(Edm.DateTimeOffset)",
-        "Collection(Edm.GeographyPoint)",
-        "Collection(Edm.ComplexType)"
-      );
-    }
+# Add static Collection<DataType> method to DataType
```

Comment:
You still need `EDM_COMPLEX_TYPE`. It is orthogonal to the `Collection` method, which should be hand-written. Refer to the .NET implementation; the Java implementation should be equivalent.

Reply:
The `EDM_COMPLEX_TYPE` is still there. We only removed the Collections of all types (or didn't add them as fixed types as before) and added the Collection() method, as done in the .Net implementation.